src/styles/contactpagebox.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "./variables.scss";
+
+.cardBackground {
```

Comment:
This is largely copied and pasted over from the WholePageBox module. Consider using making use of mixins.